### ISSUE TITLE
Google Lighthouse / accessibility test runnen

### ISSUE THREAD

**Issue by Lmikkers** [fdnd-agency/buurtcampus-oost#140]
### Story 2
De website van bieb in bloei optimaliseren

### User story
Als bezoeker wil ik mij geen zorgen hoeven te maken over of ik wel door de pagina’s heen kan als slechtziende/met een fysieke beperking.

### Taak
Google Lighthouse / accessibility test runnen

**Comment by wingsvn** [comment]
**Google Lighthouse**
Ik heb de Google Lighthouse test laten runnen op de huidige bieb-in-bloei live website. Hier uit kwam een score van 99.

![Image](https://github.com/user-attachments/assets/bfac2dae-2b01-4467-96c5-f22714c6862e)

problemen die uit de test kwamen zijn:
* image elements have [alt] attributes that are redundant text https://dequeuniversity.com/rules/axe/4.8/image-redundant-alt

**A11Y accessibility test*

**Comment by wingsvn** [comment]
** A11Y accessibility test**

Content
* [x] Use plain language and avoid figures of speech, idioms, and complicated metaphors.
* [ ] Make sure that `button`, `a`, and `label` element content is unique and descriptive.

> Op de homepage van bieb-in-bloei hebben alle links de beschrijving "bekijk de bieb". We zouden dit kunnen veranderen naar iets meer specifieks, bijv. " bekijk de stekjes" of bekijk de zaden". Ook fijn voor mensen met screenreaders.
> Op de agenda pagina geldt hetzelfde.

* [x] Use left-aligned text for left-to-right (LTR) languages, and right-aligned text for right-to-left (RTL) languages.

_**Global Code**_
* [ ] Validate your HTML.
>  This document appears to be written in Dutch but the html start tag has lang="en". Consider using lang="nl" (or variant) instead.

* [ ] Use a lang attribute on the html element.
>  Is aanwezig maar moet veranderd worden naar "nl" ipv "en".

* [x] Provide a unique title for each page or view.
* [x] Ensure that viewport zoom is not disabled.
* [x] Use landmark elements to indicate important content regions.
* [x] Ensure a linear content flow.
* [x] Avoid using the autofocus attribute.
* [x] Allow extending session timeouts.
* [x] Remove title attribute tooltips.

**_Keyboard_**
zie #131 

**_Images_**
* [x] Make sure that all img elements have an alt attribute.
* [ ] Make sure that decorative images use null alt (empty) attribute values.
* [x] Provide a text alternative for complex images such as charts, graphs, and maps.
* [x] For images containing text, make sure the alt description includes the image's text.

**_Headings_**
* [x] Use heading elements to introduce content.
* [x] Use only one h1 element per page or view.
* [x] Heading elements should be written in a logical sequence.
* [x] Don't skip heading levels.

**_Color contrast_**
* [x] Check the contrast for all normal-sized text.
* [x] Check the contrast for all large-sized text.
* [x] Check the contrast for all icons.
* [ ] Check the contrast of borders for input elements (text input, radio buttons, checkboxes, etc.).
* [ ] Check text that overlaps images or video.
* [ ] Check custom ::selection colors.

Het kleurencontrast van de tekst met de achtergrond op de bieb-in-bloei pagina voldoet grotendeels aan de WCAG richtlijnen. 

<img width="955" alt="image" src="https://github.com/user-attachments/assets/e866ff14-baaa-4b54-aa34-945b137a209b">

<img width="476" alt="image" src="https://github.com/user-attachments/assets/7708c81b-f465-4e2f-90b9-38734c575cac">

<img width="962" alt="image" src="https://github.com/user-attachments/assets/5936130e-8483-4066-831f-19387ba173ef">

<img width="470" alt="image" src="https://github.com/user-attachments/assets/9c016a9b-601d-43d3-9b01-cdd1dc74a56c">

<img width="914" alt="image" src="https://github.com/user-attachments/assets/8f6735b9-665e-4b55-9be2-54c1d1cfe4ed">

<img width="923" alt="image" src="https://github.com/user-attachments/assets/c037841a-c0a9-4a91-874e-c0ed11490c66">

<img width="923" alt="image" src="https://github.com/user-attachments/assets/24d8eae9-8038-4929-847b-4e39847cbc47">

Hieronder een aantal punten waar het contrast niet voldoende is:
<img width="940" alt="image" src="https://github.com/user-attachments/assets/c02d0fa6-155a-4f74-bceb-7e65589734f7">
witte tekst op een lichte achtergrondfoto

<img width="659" alt="image" src="https://github.com/user-attachments/assets/8cdd37f6-abf8-438d-a5c6-b9956a1cf634">
onvoldoende contrast van de tekst/link en de achtergrond

<img width="877" alt="image" src="https://github.com/user-attachments/assets/9582df4d-4e3e-49ef-b75f-ab714f84297a">
contrast van de border van het input veld en de achtergrond is niet voldoende


**_Font-size_**
De font-size in het input veld van het contactformulier en in de tekst onder aan de footer is kleiner dan 16px. Niet voor iedereen makkelijk leesbaar.

<img width="942" alt="image" src="https://github.com/user-attachments/assets/9727bd61-41b8-4410-8069-248bfea30c0e">

<img width="502" alt="image" src="https://github.com/user-attachments/assets/4bb9a3f1-d246-48a3-89c8-2e33c6484042">
